layers/+lang/purescript/layers.el

```diff
@@ -9,4 +9,6 @@
 ;;
 ;;; License: GPLv3
 
-(configuration-layer/declare-layer-dependencies '(node))
```

Comment:
Removing the dependency on the node layer will probably break part of the layer code, please keep this.